apps/aechannel/src/aesc_listeners.erl

```diff
@@ -103,18 +109,23 @@ handle_call({listen, Port, Responder, Opts}, {Pid,_Ref}, #st{ ports = Ports
                     {reply, {exception, Reason}, St}
             end
     end;
-handle_call({close, LSock}, {Pid,_Ref}, #st{ socks = Socks
+handle_call({close, LSock}, {Pid,_Ref}, #st{ responders = Resps
```

Comment:
Who calls `aesc_listeners:close/1`? Nobody? Shall this piece of API be deleted?

Reply:
> Who calls `aesc_listeners:close/1`? Nobody? Shall this piece of API be deleted?

I'm undecided. It seems symmetric to have it. Ok if we keep it around for a while? It might come in handy.